SABnzbd.py

```diff
@@ -1169,10 +1169,18 @@ def main():
         logging.info('Platform = %s', os.name)
     logging.info('Python-version = %s', sys.version)
     logging.info('Arguments = %s', sabnzbd.CMDLINE)
+
+    # Find encoding; relevant for unrar activities
     try:
-        logging.info('Preferred encoding = %s', locale.getpreferredencoding())
+        preferredencoding = locale.getpreferredencoding()
+        logging.info('Preferred encoding = %s', preferredencoding)
     except:
         logging.info('Preferred encoding = ERROR')
+        preferredencoding = ''
+    # On Linux/FreeBSD/Unix "UTF-8" is strongly, strongly adviced:
+    if 'UTF-8' not in preferredencoding and not sabnzbd.WIN32 and not sabnzbd.DARWIN:
```

Comment:
I searched for the possible return values, but it's not clear. It's probably also OS dependent.

How about: `preferredencoding.lower().find('utf') >= 0`

```
$ python
Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
[GCC 6.3.0 20170118] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> preferredencoding = 'UTF-8'
>>> print preferredencoding.lower().find('utf') >= 0
True
>>> 
>>> preferredencoding = 'UTF-16'
>>> print preferredencoding.lower().find('utf') >= 0
True
>>> 
>>> preferredencoding = 'something with utf'
>>> print preferredencoding.lower().find('utf') >= 0
True
>>> 
>>> preferredencoding = 'ANSI XYZ'
>>> print preferredencoding.lower().find('utf') >= 0
False
>>> 
```